application/org.openjdk.jmc.ui/src/main/java/org/openjdk/jmc/ui/misc/ChartTextCanvas.java

```diff
@@ -67,7 +67,9 @@
 import org.openjdk.jmc.ui.misc.PatternFly.Palette;
 
 public class ChartTextCanvas extends Canvas {
-	private static int MIN_LANE_HEIGHT = 50;
+	private static final int DEFAULT_LANE_HEIGHT = 50;
```

Comment:
Not sure how I feel about these two variables having the same name .. maybe the `final` one could be the `DEFAULT` and the practical one be just `LANE_HEIGHT`? Or something similar to that?

Reply:
Hmm what two variables are you talking about? I set the laneHeight variable to be initialised to the DEFAULT_LANE_HEIGHT value - is this what you are reffering to ?

Reply:
Hm disregard that, I think my browser (or eyes) were playing tricks on me when I read through it the first time.